### PR TITLE
Several minor fixes and fixing SPARQL queries accuracy and removing (again) duplicates.

### PR DESCRIPTION
**Pull request by olea**

### CODE REVIEW

File: components/WLMonumentProperties.vue [hunk 1]
```diff
@@ -2,32 +2,26 @@
...
-        <div v-if="prop.propertyId === 'P17'" class="row">
-          <h3 class="categoryName">
-            País:
-          </h3>
-          <p class="propValue">
-            {{ prop.value }}
-          </p>
-        </div>
-        <div v-else-if="prop.propertyId === 'P131'" class="row">
+        <div v-if="prop.propertyId === 'P131'" class="row">
```
Comment (njrzr) [inline components/WLMonumentProperties.vue:7]: @olea please fix the linter warnings, this is 1 of 2 warnings.

File: components/WLMonumentProperties.vue [hunk 2]
```diff
@@ -2,32 +2,26 @@
...
         <div v-else-if="prop.propertyId === 'P149'" class="row">
           <h3 class="categoryName">
-            Categorías:
+            Estilo:
           </h3>
           <p class="propValue">
             {{ prop.value }}
           </p>
         </div>
+<!--
```
Comment (njrzr) [inline components/WLMonumentProperties.vue:24]: @olea please fix the linter warnings, this is 2 of 2 warnings.

**Review by njrzr — APPROVED** [review]

File: components/WLMonumentProperties.vue (hunk 1 above)
Comment (olea) [reply components/WLMonumentProperties.vue:7]: done

File: components/WLMonumentProperties.vue (hunk 2 above)
Comment (olea) [reply components/WLMonumentProperties.vue]: done.

**Review by njrzr — APPROVED** [review]

**Pull request merged**